packages/devextreme/testing/testcafe/tests/dataGrid/keyboardNavigation/startEditing.functional.ts

```diff
@@ -35,3 +35,42 @@ test('Editing should start by pressing enter after scrolling content with scroll
     height: 300,
   });
 });
+
+test('DataGrid should not remove the minus symbol when editing started (T1201166)', async (t) => {
+  const dataGrid = new DataGrid(DATA_GRID_SELECTOR);
+
+  await t
+    .click(dataGrid.getDataCell(0, 0).element)
+    .pressKey('- 1')
```

Comment:
Question.

Is this space between ```-``` and ```1``` expected here?

Reply:
Yep. It is pressKey method. The space is not required in the typeText.